src/service/blockChainManager.ts

```diff
@@ -35,3 +35,10 @@ export const mine = (bloc:block, difficulty:number) : block => {
 
 export const addAblockToBlockChain = (blockchain:block[]) :block[] => (
   [...blockchain, mine(getLastBlock(blockchain), 2)]);
+
+export interface voter{
+  privateKey:string;
```

Comment:
you should move this interface to /src/model/voter ,
the others interfaces should be in src/model/filename too